src/django/oar/settings.py

```diff
@@ -334,6 +334,7 @@
 
 # Application settings
 MAX_UPLOADED_FILE_SIZE_IN_BYTES = 5242880
+TILE_CACHE_MAX_AGE_IN_SECONDS = 60 * 60 * 24
```

Comment:
I'm OK with this number too. I may have started with something lower (5-60 minute range), but in the end we'll have to evaluate based on the cache hit rates we see on the distribution as real traffic hits the tile endpoint. Important part is that we have a non-TTL based way to bust the cache.

Reply:
We can bust the cache by incrementing a "version" column in the database.

Reply:
Since allow query forwarding in Cloudfront configuration, we can also bust the cache using query strings.

Reply:
(But yes in practice the Cache Key which is a function of database version number will likely be used to request a new URI for the same tile)